### PR TITLE
Fix permission in certificates

### LINKED ISSUE #602: The indexer/dashboard certs folder should have 500 permissions and certificates 400
The permissions given to `certs` folder (the certificates containing folder) should change to 500, which is more restrictive. Certificates should have 400.

### PR DESCRIPTION
**Pull request by fcaffieri**
|Related issue|
|---|
|This PR closes #602|

This pr modifies the permissions of the Indexer/Dashboard certs folder to 500 permissions and certificates 400.

It also fixes the problem that the manager's root-ca differs in permissions from the indexer's and dashboard's. A change has been implemented so that it places permissions 1000:1000 for indexer and dashboard and 999:997 for manager and worker.

image
image
image
image

### CODE REVIEW

**Review by vcerenu — APPROVED** [review]
LGTM

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the owner problem, the stack doesn't spin up.

File: build-docker-images/wazuh-indexer/config/config.sh [hunk 1]
```diff
@@ -104,3 +104,6 @@ cp -pr /wazuh-certificates/root-ca.key ${TARGET_DIR}${CONFIG_DIR}/root-ca.key
 cp -pr /wazuh-certificates/root-ca.pem ${TARGET_DIR}${CONFIG_DIR}/root-ca.pem
 cp -pr /wazuh-certificates/admin.pem ${TARGET_DIR}${CONFIG_DIR}/admin.pem
 cp -pr /wazuh-certificates/admin-key.pem ${TARGET_DIR}${CONFIG_DIR}/admin-key.pem
+
+chmod -R 500 ${TARGET_DIR}${CONFIG_DIR}
+chmod -R 400 ${TARGET_DIR}${CONFIG_DIR}/*
```
Comment (okynos) [inline build-docker-images/wazuh-indexer/config/config.sh:109]: Why the files aren't inside certs folder?

**Review by okynos — CHANGES_REQUESTED** [review]
Minor request

**Comment by fcaffieri** [comment]
### Change directory of dashboard certs

####Test's
image
image
image
image

File: build-docker-images/wazuh-indexer/config/config.sh (hunk 1 above)
Comment (fcaffieri) [reply build-docker-images/wazuh-indexer/config/config.sh]: Done [d927698](https://github.com/wazuh/wazuh-docker/pull/604/commits/d9276985c3ea8165610b749cf00f29ee44609757)

**Review by vcerenu — DISMISSED** [review]
LGTM

**Review by okynos — APPROVED** [review]
LGTM

**Review by alberpilot — CHANGES_REQUESTED** [review]
Tested with failure:
```
docker-compose -f generate-indexer-certs.yml run --rm generator         
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 24811  100 24811    0     0  32949      0 --:--:-- --:--:-- --:--:-- 32949
Cert tool exists in Packages-dev bucket
mv: cannot move '/tmp/wazuh-install-files/new_config.yml' to '//config.yml': Device or resource busy
07/04/2022 13:05:16 INFO: Admin certificates created.
07/04/2022 13:05:16 INFO: Wazuh indexer certificates created.
07/04/2022 13:05:17 INFO: Wazuh server certificates created.
07/04/2022 13:05:17 INFO: Wazuh dashboard certificates created.
Moving created certificates to destination directory
changing certificate permissions
Setting UID indexer and dashboard
Setting UID for wazuh manager and worker

NAME
        wazuh-cert-tool.sh - Manages the creation of certificates of the Wazuh components.

SYNOPSIS
        wazuh-cert-tool.sh [OPTIONS]

DESCRIPTION
        -a,  --admin-certificates </path/to/root-ca.pem> </path/to/root-ca.key>
                Creates the admin certificates, add root-ca.pem and root-ca.key.

        -A, --all </path/to/root-ca.pem> </path/to/root-ca.key>
                Creates Wazuh server, Wazuh indexer, Wazuh dashboard, and admin certificates. Add a root-ca.pem and root-ca.key or leave it empty so a new one will be created.

        -ca, --root-ca-certificates
                Creates the root-ca certificates.

        -v,  --verbose
                Enables verbose mode.

        -wd,  --wazuh-dashboard-certificates </path/to/root-ca.pem> </path/to/root-ca.key>
                Creates the Wazuh dashboard certificates, add root-ca.pem and root-ca.key.

        -wi,  --wazuh-indexer-certificates </path/to/root-ca.pem> </path/to/root-ca.key>
                Creates the Wazuh indexer certificates, add root-ca.pem and root-ca.key.

        -ws,  --wazuh-server-certificates </path/to/root-ca.pem> </path/to/root-ca.key>
                Creates the Wazuh server certificates, add root-ca.pem and root-ca.key.
```

**Review by vcerenu — APPROVED** [review]
LGTM

**Review by vcerenu — DISMISSED** [review]

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**